wasm/wasm-introduction.md

```diff
@@ -0,0 +1,157 @@
+for a **stack-based virtual machine**. We know this definition is almost the same shown at [webassembly.org](https://webassembly.org/)
+so let extend a little bit more this explanation. Let's start with **binary instruction format**, this one is simple
+because it represents the way [WASM](https://webassembly.org/) exposes instructions to the computer to be processed. On
+the other hand, we have a **stack-based virtual machine** which stores the instructions from WASM files in memory
+using a stack as the main structure(We will talk more about this later in this post).
+Now that we are understanding a little bit more the concept of WebAssembly, so let's continue with the goals and benefits this technology brings to us.
+### Main goals of WebAssembly
+Like any other technology on the web, WebAssembly has been created thinking about some specific goals in mind. We are
+presenting these goals differently as follows:
+- Security: WebAssembly provides a secure sandboxed virtual machine. This means that we can WASM code from any web application
+without thinking about intrusions over our personal computers or servers. Proof of this point is that some of the most used browsers
+on the web have its support enabled by default because they are confident about the security provided by [WASM](https://webassembly.org/).
+- Fast execution: [WASM](https://webassembly.org/) is compiled into a binary format that will be executed at a low level. In this way, we will deliver
+near-native performance execution times. The format used by WASM is very compact so the browser can parse it as fast as possible.
+The translation of a [WebAsembly](https://webassembly.org/) module to a native code can be cached by the browser. In this way, the
+next time we load a page that contains a module, there will be virtually no load time.
+- Portability: Three of the promises of WASM are Hardware/Language/Platform independence. This principle implies portability
+for these three different factors. In the example presented in this post, we are generating a module made with ```wat2wasm``` tool in one
+folder of our system. Then we can use the same *.wasm file in our ClojureScript project without any problem at all.
+A valuable note for the reader could be a list of different languages that compile to WebAssembly, if you want this resource, please go to this [link](https://github.com/appcypher/awesome-wasm-langs).
+### The future!
+[WebAssembly](https://webassembly.org/) is one of the most exciting technologies to learn on the web at the moment and the main
+reason for this is the bigger ambition around being an alternative output format for any compiler that targets native code. 
+Another aspect to take into consideration is that [WASM](https://webassembly.org/) is already integrated with a great part of the web 
```

Comment:
Not sure about "with a great part of the web platform." Maybe just "a greater part of the web."